openpype/hosts/flame/startup/openpype_in_flame.py

```diff
@@ -3,16 +3,16 @@
 from Qt import QtWidgets
 from pprint import pformat
 import atexit
-import openpype
+
 import avalon
 import openpype.hosts.flame.api as opfapi
+from openpype.pipeline import install_host
 
 
 def openpype_install():
     """Registering OpenPype in context
     """
-    openpype.install()
-    avalon.api.install(opfapi)
+    install_host(opfapi)
```

Comment:
Is this missing an `install_openpype_plugins()`?

Reply:
Resolved with [clarified comment here](https://github.com/pypeclub/OpenPype/pull/3009#discussion_r841740121)